earth_enterprise/Upgrade.md

```diff
@@ -0,0 +1,20 @@
+# Upgrading Earth Enterprise Fusion and Server to version 5.2.1
+
+Upgrading to GEE 5.2.1 is supported from Earth Enterprise 5.2.0 version using the following steps:
+
+1. Do NOT uninstall GEE 5.2.0. We recommend that you upgrade GEE 5.2.0 by simply installing GEE 5.2.1. Installing GEE 5.2.1 on top of GEE 5.2.0 will ensure that your PostgreSQL databases are backed up and upgraded correctly to the new PostgreSQL version used by GEE 5.2.1. [Install Fusion or Earth Server](https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server). This can be achieved by running the scripts to install fusion and earth server:
+
+```bash
+cd earth_enterprise/src/installer
+sudo ./install_fusion.sh 
+sudo ./install_server.sh
+```
+
+Alternatively, instead of using scripts, on Redhat and Centos, you can upgrade GEE 5.2.0 to GEE 5.2.1 using [RPM packages] (https://github.com/google/earthenterprise/blob/master/earth_enterprise/BUILD_RPMS.md)
```

Comment:
Style: 'Redhat' -> 'Red Hat', 'Centos' -> 'CentOS'.

Typo: Remove space between `[]` and `()` in Markdown link. Otherwise, it won't render as a link.